WPF/MainWindow.xaml.cs

```diff
@@ -61,63 +69,121 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
+            _sorter = new Sorter(OriginalList.Cast<Item>().ToList());
+        }
+
+        private void dynamicProcessorCheckBox()
+        {
+            int column = 1;
+            int cycleCount = 1;
+            foreach (var processor in processors)
+            {
+                CheckBox checkbox = new CheckBox()
+                {
+                    Content = processor,
+                    Name = processor.Replace(" ", ""),
+                    FontFamily = new FontFamily("Candara Light"),
+                    Background = Brushes.White,
+                    BorderBrush = Brushes.White,
+                    Foreground = Brushes.White
+                };
+                brandsCheckBoxes.Add(checkbox);
+                switch (column)
+                {
+                    case 1:
+                        ProcessorColumn1.Children.Add(checkbox);
+                        column = 2;
+                        break;
+                    case 2:
+                        ProcessorColumn2.Children.Add(checkbox);
+                        column = 1;
+                        break;
+                    case 3:
+                        ProcessorColumn3.Children.Add(checkbox);
+                        column = 4;
+                        break;
+                    case 4:
+                        ProcessorColumn4.Children.Add(checkbox);
+                        column = 3;
+                        break;
+                }
+                cycleCount++;
+                if (cycleCount == 7) column = 3;
+            }
+        }
+
+        private void DynamicBrandCheckBox()
+        {
+            int column = 1;
+            int cycleCount = 1;
+            foreach(var brand in brands)
+            {
+                CheckBox checkbox = new CheckBox()
+                {
+                    Content = brand,
+                    Name = brand,
+                    FontFamily = new FontFamily("Candara Light"),
+                    Background = Brushes.White,
+                    BorderBrush = Brushes.White,
+                    Foreground = Brushes.White
+                };
+                brandsCheckBoxes.Add(checkbox);
+                switch(column)
+                {
+                    case 1:
+                        BrandColumn1.Children.Add(checkbox);
+                        column = 2;
+                        break;
+                    case 2:
+                        BrandColumn2.Children.Add(checkbox);
+                        column = 1;
+                        break;
+                    case 3:
+                        BrandColumn3.Children.Add(checkbox);
+                        column = 4;
+                        break;
+                    case 4:
+                        BrandColumn4.Children.Add(checkbox);
+                        column = 3;
+                        break;
+                }
+                cycleCount++;
+                if (cycleCount == 7) column = 3;
+            }
```

Comment:
this code has many same code lines. Maybe it is possible to make new method for it so there won't be any repetitions?